### ISSUE TITLE
how to specify a context root?

### ISSUE THREAD

**Issue by wgorder** [Swiip/generator-gulp-angular#290]
With the inject stuff is there a way to prefix the injected paths with an application context root?

I have tried using the `addPrefix` option for that but unfortunately it adds it to the file system path as its searching for the file, so I get an error.

**Comment by wgorder** [comment]
Any suggestions on this?  I tried using an inject transform to add the context root to the URL when it is injected in index.html but it broke all sort of stuff since it also looks for that as the file path.

**Comment by Swiip** [comment]
The context question is still an open issue. I don't say it's impossible I just never got the chance to work on the question... But I will

**Comment by wgorder** [comment]
Thanks @Swiip .  I think I got it mostly working I have some more testing to do with it.  Unfortunately it required a lot of changing to paths all over the place.  I know there is probably a better way, I will wait to see what you come up with.

**Comment by wgorder** [comment]
This one is still really causing me some pain.  If someone wants to give me a hint as to the best way to look at implementing this, I would be willing to look at working on it.

I have looked at the inject, and wiredep stuff some.  As well as just adding another nested directory by editing those in index.html.  All of this seems to half way work or not work well though.

**Comment by sebastianovide** [comment]
Hello, wgorder, I was considering about doing the same as one of my UI devs has evolved a nice template using millions of jquery plugins and we want to use it as landing page + login. So the angular app will go in /webapp. 
But thinking twice, probably it is better to keep them as separate projects and just deploy them on separate folders.
What do you think ?

cheers

**Comment by zckrs** [comment]
Can you retry with latest version of generator please ?

**Comment by wgorder** [comment]
What changed in the new generator?  I had ended up using the cheerio solution in #419 

Anyway I have moved toward a more component based approach to angular without bower.  I've diverged to the point where I have created a different seed project that really looks nothing like this one anymore.  It starts with https://github.com/angular-class/NG6-starter and adds a lot of the missing functionality that this generator does have that I like e.g. cache busting, source maps etc.  Still a WIP but so different at this point, that I am not sure how I would contribute it here.
